CHANGELOG.md

```diff
@@ -4,6 +4,35 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.18.0
+
+**Released**: 2021.09.13
+
+**Commit Delta**: [Change from 0.17.1 release](https://github.com/plus3it/tardigrade-ci/compare/0.17.0..0.18.0)
+
+**Summary**:
+
+*   The new environment variable "PROVIDER_ALIAS" can be used to specify the 
+    name of a Terraform provider alias.  Used as a value for the new "--alias" 
+    command line option, it instructs the automated integration test tool
+    to create the aliased provider information in preparation for a test run.
+
+*   When running Terraform tests, the Firehose service will now be provided
+    by moto and not LocalStack.
+
+*   No explicit changes to tool versions, but dependabot changes since prior 
```

Comment:
We can just say:

```suggestion
*   Updates tool versions:
```

This is documenting the release, not the changes in this particular branch...

Reply:
We can hold off on listing all these out until localstack updates and we're ready to release and we update .bumpversion.cfg. These tools/versions may change between now and then anyway.

Reply:
```suggestion
*   Updates tool versions:
```

Reply:
Also, these versions aren't quite right, since the branch is out-of-date with master it is more difficult to setup the compare to see the correct versions. Updating the branch should make it easier to see the real diff against the last tag.